ignite/contrib/handlers/base_logger.py

```diff
@@ -161,12 +161,20 @@ def attach(
         Returns:
             :class:`~ignite.engine.RemovableEventHandle`, which can be used to remove the handler.
         """
-        name = event_name
+        if isinstance(event_name, EventsList):
+            for name in event_name:
+                if name not in State.event_to_attr:
+                    raise RuntimeError(f"Unknown event name '{name}'")
```

Comment:
this line is not covered by tests

Reply:
You are right even if it implies hacking `EventsList` in order to raise this exception. Btw python so permissive that I add it !!

That you for this catch !